moped-editor/src/views/projects/projectView/ProjectSummaryMap.js

```diff
@@ -70,6 +71,8 @@ const ProjectSummaryMap = ({
    * Let's throw an error intentionally if there are no features for a project.
    */
   if (featureCount < 1) {
+    stopReportingRuntimeErrors();
```

Comment:
i am so happy about this, i have been commenting out these lines when i develop locally.